docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-agent-files-service.md

```diff
@@ -0,0 +1,121 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Configure the agent with file sources
+
+In order for Moderne to obtain information about your organizational structure, you will need to create some configuration files and direct the Moderne agent to said files. This guide will explain how to do both of those things.
+
+## repos.csv
+
+This is a required file, which outlines the repositories and the organizations which they are associated with.
+When provided your organization details, they must be provided in a CSV file with the following format:
+
+| cloneUrl      | branch   | org1    | org2        | org3 |
+|---------------|----------|---------|-------------|------|
+| `https://github.com/openrewrite/rewrite-recipe-bom` | main | Open Source | ALL | |
+| `https://github.com/Netflix/spectator-go` | main | Netflix | Open Source | ALL |
+
+The organizations under `org1`, `org2`, `org3`, etc. represent the hierarchy of organizations. There is no limit to the number of organizations that can be provided via this CSV.
+
+The above example would be used in Moderne DX to generate an organizational listing of the following:
+
+* ALL
+    * Open Source
+        * `https://github.com/openrewrite/rewrite-recipe-bom:main`
+        * Netflix
+            * `https://github.com/Netflix/spectator-go:main`
+
+To generate this `repos.csv`, we recommend using "[repo fetchers](https://github.com/moderneinc/repository-fetchers)".
+
+## commitOptions.txt
+
+This is an optional file, which allows for custom commit options for individual repositories. When no commit options are provided for a specific repositories we will fall back the default commit options which are specified in the [agent configuration](agent-variables.md#all-agent-configuration-variables) or all commit options if no default are provided.
```

Comment:
Do you think people will understand what a commit option is? I wonder if we should add something like: 

"Commit options let you configure how code changes can be committed. For example, you could specify that you only want your organization to allow pull requests. Or you could allow people to commit the results directly to main."

This could come after the first sentence.

Reply:
I think adding a sentence to make sure it clear is a good, idea. I also found a like outlining all the options which are available and I included it